server/views/pages/changeTeam.njk

```diff
@@ -11,6 +11,13 @@
         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full">
+                {% set probationTeamOptions = [] %}
+                {% for team in probationTeamsWithCount %}
+                    {% set probationTeamOptions = (probationTeamOptions.push(
+                        {   value: team.code, 
+                            html: '<p class="govuk-!-margin-0">'+ team.label  +' (<b>'+ team.count +'</b> licences to review)</p>' if showTeamsCount else '<p class="govuk-!-margin-0">'+ team.label +'</p>'
```

Comment:
can we wrap the `team.label` with something to sanitise any html?

Reply:
` | escape` around both should work

Reply:
do you need to set the checked attribute?

Reply:
yes, I completely missed it.